### DISCUSSION TITLE
How can i get a databricks output from create_databricks_job_solid

### DISCUSSION THREAD

**Discussion by ericholiveira** [dagster-io/dagster#3367] [Q&A]
I'm testing dagster and so far think it is a great tool. 
In one of my tests im running jobs in databricks using create_databricks_job_solid. The problem is that this solid doesnt return any info about the job run (except for success and failure default messages) it doesnt even return the job_id/run_id so i can check in dataabricks UI.

Am i missing anything? Or this is not implemented yet? How are people checking the results of their jobs?

**Comment by sryza** [comment]
Hi @ericholiveira - this should be included in the logs:

```
        context.log.info(
            "Launched databricks job with run id {run_id}. UI: {url}. Waiting to run to completion...".format(
                run_id=run_id, url=create_ui_url(databricks_client, context.solid_config)
            )
        )
```

Would you like this the run_id to be included in the object that's the output of the solid?  That seems like it shouldn't be too difficult.

**Comment by ericholiveira** [comment]
Thanks for the response @sryza  the problem is that create_ui_url logs one of the following url:
- "https://{host}/?o={workspace_id}#/setting/clusters/{cluster_id}/sparkUi"
- "https://{host}/?o={workspace_id}#joblist"

And from none of that i can see the output from the job run (actually idk if it is a bug in databricks, but i cant even see the job created through  create_databricks_job_solid in databricks UI)

To see the output of the job run i need an url as follow:
https://{host}/?o={workspace_id}#job/{job_id}/run/{run_id}

this isn't hard to implement, actually with two more calls to the api on create_databricks_job_solid  we can get that... or even if at least create_databricks_job_solid returned the run_id i could call the api and get the url by myself

**Comment by catherinewu** [comment]
Hi @ericholiveira, thanks for the question! seems like a very reasonable thing for create_databricks_job_solid to support. tracking the feature request here: https://github.com/dagster-io/dagster/issues/3495
